### DISCUSSION TITLE
how to get fi.get_farm_power for no wake conditions

### DISCUSSION THREAD

**Discussion by Teddybear0227** [NatLabRockies/floris#792] [Q&A]
Hello,

I am trying to quantify the wake effect on wind farm power by finding the difference in power output vs wind direction with wake effect and without wake effect. 

The FLORIS version I am using does not allow fi.calculate_wake(no_wake=True). 

Is there any other methods which I can use fi.get_farm_power for no wake conditions? If the only option is to update FLORIS, how do I update? (I have a few turbine data inputs which I do not want to lose because of the update!!)

Sincerely,

**Comment by rafmudaf** [comment]
@Teddybear0227 which version of FLORIS are you currently using? You can check with these commands in your terminal:
```bash
$ python
Python 3.10.4 (main, Mar 31 2022, 03:38:35) [Clang 12.0.0 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import floris
>>> floris.__version__
'3.5'
```

**Reply by Nellytado** [reply]
Hi, I am trying to do this using Floris 3.6. I am trying to quantify the wake effect on wind farm power by finding the difference in power output vs wind direction with wake effect and without wake effect. The examples 4-6 do not really show how to generate power output across wind directions with and without wake. This is for a student project your help is most appreciated!

**Comment by paulf81** [comment]
I think this is also discussed in #895 so just wanted to link
